### PR TITLE
[Crossdata-138] [Driver] Server and Driver Distribution 

### PR DESCRIPTION
**Pull request by luismcl**

### CODE REVIEW

File: driver/pom.xml [hunk 1]
```diff
@@ -129,5 +129,334 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>package</id>
+            <build>
+                <plugins>
+                    <!--<plugin>-->
```
Comment (darroyocazorla) [inline driver/pom.xml]: Remove the plugin

File: driver/pom.xml [hunk 2]
```diff
@@ -129,5 +129,334 @@
...
+                                </goals>
+                                <configuration>
+                                    <architecture>noarch</architecture>
+                                    <assembly>
+                                        <copyFile>
+                                            <path>src/main/dist/config/driver-application.conf</path>
+                                            <toFile>/etc/sds/crossdata/driver/driver-application.conf</toFile>
+                                        </copyFile>
+                                        <copyFile>
+                                            <path>src/main/dist/config/core-application.conf</path>
```
Comment (darroyocazorla) [inline driver/pom.xml]: Remove the core-application.conf

File: driver/pom.xml [hunk 3]
```diff
@@ -129,5 +129,334 @@
...
+                                <configuration>
+                                    <architecture>all</architecture>
+                                    <assembly>
+                                        <copyFile>
+                                            <path>src/main/dist/config/driver-application.conf</path>
+                                            <toFile>/etc/sds/crossdata/driver/driver-application.conf</toFile>
+                                        </copyFile>
+                                        <copyFile>
+                                            <path>src/main/dist/config/core-application.conf</path>
+                                            <toFile>/etc/sds/crossdata/driver/core-application.conf</toFile>
```
Comment (darroyocazorla) [inline driver/pom.xml]: Same as above

File: driver/pom.xml [hunk 4]
```diff
@@ -129,5 +129,334 @@
...
+                                        </copyFile>
+                                        <copyFile>
+                                            <path>src/main/dist/config/xdmetrics.properties</path>
+                                            <toFile>/etc/sds/crossdata/driver/xdmetrics.properties</toFile>
```
Comment (darroyocazorla) [inline driver/pom.xml]: Remove xdmetrics

File: driver/pom.xml [hunk 5]
```diff
@@ -129,5 +129,334 @@
...
```
Comment (darroyocazorla) [inline driver/pom.xml]: Same as above

File: driver/src/main/dist/config/core-application.conf [hunk 6]
```diff
@@ -0,0 +1,7 @@
+################################
```
Comment (darroyocazorla) [inline driver/src/main/dist/config/core-application.conf]: Remove file

File: server/pom.xml [hunk 7]
```diff
@@ -113,52 +113,307 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-shade-plugin</artifactId>
-                        <version>2.4.1</version>
+                        <groupId>org.codehaus.mojo</groupId>
```
Comment (darroyocazorla) [inline server/pom.xml]: We should test whether it works in a distributed environment or not after removing the dependencies.

Comment (darroyocazorla) [inline server/pom.xml]: We cannot remove the transformers which add the META-INF necessary for SparkSQL and Crossdata

**Comment by miguel0afd** [comment]
:+1:

**Pull request closed without merging**